src/pages/index.tsx

```diff
@@ -0,0 +1,311 @@
+import React, { useRef } from 'react';
+import OriginalLayout from '@theme/Layout';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUsers, faRocket } from '@fortawesome/free-solid-svg-icons';
+import { faGithub, faTwitter } from '@fortawesome/free-brands-svg-icons';
+import ProductCard from '../components/ProductCard';
+import ScrollAnimationWrapper from '../components/ScrollAnimationWrapper';
+import clsx from 'clsx';
+import Link from '@docusaurus/Link';
+
+const FAIcon = FontAwesomeIcon as unknown as React.ComponentType<any>;
+const DocusaurusLink = Link as unknown as React.ComponentType<any>;
+
+const ThemedLayout = OriginalLayout as unknown as React.ComponentType<any>;
+
+
+const products = [
+  {
+    title: "Phoenix",
+    description: "The ultimate network management suite for security, ranks, and staff tools.",
+    href: "/Phoenix/Introduction",
+    
+    colorClass: "bg-red-600",
+    logoSrc: "/logo.png",
+  },
+  {
+    title: "Bolt",
+    description: "High-performance PvP practice plugin with advanced ELO and match tracking.",
+    href: "/Bolt/Introduction",
+    
+    colorClass: "bg-yellow-600",
+    logoSrc: "/logo.png",
+  },
+  {
+    title: "CarbonSpigot",
+    description: "Optimized Minecraft server fork for superior performance and stability.",
+    href: "/CarbonSpigot/Introduction",
+    
+    colorClass: "bg-gray-600",
+    logoSrc: "/logo.png",
+  },
+  {
+    title: "Zephyr",
+    description: "Lightweight and efficient hub management system.",
```

Comment:
Zephyr is not a hub plugin :P
```suggestion
    description: "Modern and lightweight FFA plugin.",
```